### ISSUE TITLE
Allow providing extra DEFAULT_TEXT_MIME_TYPES

### ISSUE THREAD

**Issue by georgebv** [jordaneremieff/mangum#275]
**Issue**: I want to serve binary files as is (not base64 encoded)
**Cause**: everything not in `DEFAULT_TEXT_MIME_TYPES` is base64 encoded by default
**Proposed solution**: allow providing additional mime types; suggested options:
  - when declaring the handler `Mangum(app, extra_text_mime_types=["image/png"])`
  - after declaring the handler `handler.add_text_mime_types(["image/png"])`

This is my current hacky solution to serve png images:

```python
from mangum import Mangum
from mangum.handlers.utils import DEFAULT_TEXT_MIME_TYPES

from app.main import app

# Monkey patch MIME types
DEFAULT_TEXT_MIME_TYPES.extend(
    [
        "image/png",
    ]
)

handler = Mangum(app)
```

**Comment by jordaneremieff** [comment]
Hi @georgebv, this is similar to a recent issue report in https://github.com/jordaneremieff/mangum/issues/270.

One of the solutions proposed was to re-introduce the `text_mime_types` configuration, if you are able to put forward a PR to do this, then I will review and merge.

**Comment by georgebv** [comment]
@jordaneremieff sounds good, I'll link it here once I get to it (~this week). Also, FYI, I am using mangum to deploy a full-blown web app on FastAPI to aws lambda.

**Comment by jordaneremieff** [comment]
Released in [0.16.0](https://github.com/jordaneremieff/mangum/releases/tag/0.16.0).
